docs/root/configuration/factory_categories.rst

```diff
@@ -0,0 +1,15 @@
+.. _factory_categories:
+
+Registered Factory Categories
```

Comment:
Nit: Would just rename this to `Factory Categories`

Reply:
done